src/lib/provenanceUtils.ts

```diff
@@ -14,6 +14,36 @@ export function updateProvenanceState(vuexState: State, label: ProvenanceEventTy
       // eslint-disable-next-line no-param-reassign
       provState.displayCharts = newProvState.displayCharts;
     }
+
+    if (label === 'Set Marker Size') {
+      // eslint-disable-next-line no-param-reassign
+      provState.markerSize = newProvState.markerSize;
+    }
+
+    if (label === 'Set Font Size') {
+      // eslint-disable-next-line no-param-reassign
+      provState.fontSize = newProvState.fontSize;
+    }
+
+    if (label === 'Set Label Variable') {
+      // eslint-disable-next-line no-param-reassign
+      provState.labelVariable = newProvState.labelVariable;
+    }
+
+    if (label === 'Set Color Variable') {
+      // eslint-disable-next-line no-param-reassign
+      provState.colorVariable = newProvState.colorVariable;
+    }
+
+    if (label === 'Set Select Neighbors') {
+      // eslint-disable-next-line no-param-reassign
+      provState.selectNeighbors = newProvState.selectNeighbors;
+    }
+
+    if (label === 'Set Directional Edges') {
+      // eslint-disable-next-line no-param-reassign
+      provState.directionalEdges = newProvState.directionalEdges;
+    }
```

Comment:
Using `else if` instead of parallel `if` statements would be good for two reasons:
1. prevents accidental insertion of code later between the statements
2. Makes clear to readers (via the if-else if-else idiom) that this is really a big "switch statement" (point 1 above is sort of a code enforcement of this idea).